src/samples/flow/compass/Makefile

```diff
@@ -1,7 +1,2 @@
 sample-$(FLOW_COMPASS_LSM303_SAMPLE) += compass-lsm303
 sample-compass-lsm303-$(FLOW_COMPASS_LSM303_SAMPLE) := lsm303.fbp
-sample-compass-lsm303-$(FLOW_COMPASS_LSM303_SAMPLE)-deps := \
-	flow/accelerometer.mod \
-	flow/compass.mod \
-	flow/console.mod \
-	flow/magnetometer.mod
```

Comment:
On Fri, 18 Sep 2015 15:03:09 -0700, Gustavo Sverzut Barbieri wrote:
> > @@ -1,7 +1,2 @@
> >  sample-$(FLOW_COMPASS_LSM303_SAMPLE) += compass-lsm303
> >  sample-compass-lsm303-$(FLOW_COMPASS_LSM303_SAMPLE) := lsm303.fbp
> > -sample-compass-lsm303-$(FLOW_COMPASS_LSM303_SAMPLE)-deps := \
> > -	flow/accelerometer.mod \
> > -	flow/compass.mod \
> > -	flow/console.mod \
> > -	flow/magnetometer.mod
> 
> hum... we still need the module, did you remove these dependencies just because we don't need them to be built before?

Ah, yes, in order to keep build dependencies correct we need to keep all
these. Guess I'll just scrap those patches.